site/src/components/RuntimeErrorState/createCtas.tsx

```diff
@@ -48,9 +48,7 @@ const ReloadAppButton = (): ReactElement => {
  * createCtas generates an array of buttons to be used with our error boundary UI
  */
 export const createCtas = (codeBlock: string[]): ReactElement[] => {
-  // REMARK: we don't have to worry about key order changing
-  // eslint-disable-next-line react/jsx-key
-  return [<CopyStackButton text={codeBlock.join("\r\n")} />, <ReloadAppButton />]
+  return [<CopyStackButton key="copy-stack-btn" text={codeBlock.join("\r\n")} />, <ReloadAppButton key="reload-app-btn" />]
```

Comment:
I know it says they won't change but I decided to add them because:
- no negative impact adding them
- good codebase hygiene to avoid using `eslint-disable`

Plus, in some edge case if React does render them in a different order, it can tell the difference based on `key`.